docs/examples/4_remote/plot_04_remote_h01.py

```diff
@@ -0,0 +1,112 @@
+"""
+H01 Dataset
+============
+
+In this notebook, you can learn how to work with the [H01 dataset](https://www.science.org/doi/10.1126/science.adk4858) using Navis. The H01 dataset contains 57,000 cells and 150 million synapses from a cubic millimeter of the human temporal cortex, which is [proofreading](https://h01-release.storage.googleapis.com/proofreading.html) in the CAVE. With this interface, you can access both a snapshot of the proofread dataset and the latest dataset using CAVEclient.
```

Comment:
The first line in the tutorial is used as description when hovering over the thumbnail - it should be a single, short sentence and it doesn't render markdown unfortunately, i.e. the link won't work. I'd suggest something like this:

```
In this notebook, you will learn how to work wit the H01 dataset.

The [H01 dataset](https:...) contains...
```